### ISSUE TITLE
has the eurorack PSU been tested?

### ISSUE THREAD

**Issue by KeLaiFu** [ghosty141/eurorack-psu#1]
looking for a beefy PSU and found yours. Before I build it and burn down my house, has it been built / tested yet?

Looks like the answer to all my prayers!

Cheers

**Comment by ghosty141** [comment]
Hi, yeah I'm running it in my rack. Haven't put many hours into it cause I'm currently not in the mood for eurorack but that has nothing to do with the hardware haha. I wanted to do some more documentation but I never did. But yeah it works and I've done some load testing.

If you actually wanna use it just a few heads up:

1. You MUST be able to test it yourself and **you** are liable if something breaks. I hope thats understandable, I'm not a professional EE (although two of my friends who are did work with me on this) and this is not a "proper" product so I just want to put this warning out there. It's your risk if you run this thing.
2, It does get hot, so if your case is rather small (mine kinda is, 3x60hp in a moog mother rack form factor) and the heat does build up a bit inside, so the modules get warm. Not like crazy hot but warm. Adding one computer case fan running super slowly or having a big case completely eliminates this. My point is, don't run this in a small case fully enclosed case with no air vents for example, it will probably work but testing is needed.

Regarding heat, so I did some basic calculations and it shouldn't overheat the voltage regulators. The design is VERY similar to the doepfer one just with a little bit of headroom added, so unless you pull all the watts it can deliver it should be pretty much the same as the doepfer one in terms of heat.

3. Just for peace of mind I'd add small resettable smd fuses to the outputs, they cost nothing and maybe avoid damage IF something goes wrong. I didn't add them cause that came to my mind too late and I didn't wanna print new boards and buy some already soldered components, but for a "v2" this would be a good change.
4. I don't provide any electrical characteristics here mostly cause I don't have the tools to properly (read as, professionally) test the thing and provide reliable figures. All I **can** say is, it works in my rack which looks like this  https://cdn.modulargrid.net/img/racks/modulargrid_2255648.jpg


Ok one last thing, there is a little price you pay if you build the PSU yourself: You don't have that peace of mind that it works for hundreds of other users so no worrying about it breaking. But that may be just me, my EE friend was kinda amused by my paranoia during testing haha, they are a different breed to software folks like me who always think everything breaks.

**Comment by KeLaiFu** [comment]
all sounds as expected, I'm not expecting a warranty and customer service!

One question regarding the heat, is that heat coming from the 2 large regulators and their heatsinks? I'm currently struggling to source the exact heatsinks that are on your version (the same as the doepfer) but I found some similar sized ones with maybe less performance.  I may look into other options for the heatsink if the original ones are already being pushed.

also, would it be possible to see a photo of the finished PSU?

**Comment by ghosty141** [comment]
> is that heat coming from the 2 large regulators and their heatsinks?

Yes, at most there are ~5-9W of heat being dumped per side which is quite a lot. Obviously running it at peak power is not recommended.

>  (the same as the doepfer)

The reason this psu looks so similar is because there are basically no other parts available. Negative LDOs kinda cap out at 1.5A, for the positive side its better but you are always limited by the negative side (3x the power on +12V isn't really gonna be useful). 

The same goes for the heatsinks, there aren't many available for the to-220 package that can handle the power. The only big change one can make is using the meanwell modules vs adding their own transformer, but the rest is pretty much a done deal if you ask me.

> also, would it be possible to see a photo of the finished PSU?

For sure. 

![Foto 17 01 24, 20 08 34](https://github.com/ghosty141/eurorack-psu/assets/17564201/9cf58f6b-f0a5-475b-80f5-579841f53977)
![Foto 24 07 23, 13 51 38](https://github.com/ghosty141/eurorack-psu/assets/17564201/e7003f17-cf66-47dc-9ed7-0eb777018f06)
![Foto 31 08 23, 13 36 12](https://github.com/ghosty141/eurorack-psu/assets/17564201/887c94c4-0b01-490d-b609-f77b117a96c3)
![Foto 31 08 23, 13 36 54](https://github.com/ghosty141/eurorack-psu/assets/17564201/3e1fbbb6-c07a-4693-ad7c-7bc85e5fbee7)
